src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -551,18 +577,28 @@ private void CheckClientTimeout()
         private static void AbortConnection(object state)
         {
             var connection = (HubConnectionContext)state;
-            try
-            {
-                connection._connectionAbortedTokenSource.Cancel();
-            }
-            catch (Exception ex)
-            {
-                Log.AbortFailed(connection._logger, ex);
-            }
-            finally
+
+            _ = InnerAbortConnection(connection);
+
+            async Task InnerAbortConnection(HubConnectionContext connection)
```

Comment:
static

Reply:
Didn't even realize that was a thing for local functions